src/general/MainMenu.cs

```diff
@@ -423,6 +455,9 @@ private void CreditsPressed()
         // Hide all the other menus
         SetCurrentMenu(uint.MaxValue, false);
 
+        // Hide Social Icons
+        GetNode<Control>("SocialMedia").Visible = false;
```

Comment:
You should use a NodePath and lookup the social media controls just once in _Ready